tests/unit/myth/Models/CIDbModelTest.php

```diff
@@ -543,10 +543,10 @@ public function testFindAsJson()
     {
         $this->model->db->shouldReceive('where')->once()->with('id', 1)->andReturn( $this->model->db );
         $this->model->db->shouldReceive('get')->once()->with('records_table')->andReturn( $this->model->db );
-        $this->model->db->shouldReceive('row_array')->once()->andReturn('fake object');
+        $this->model->db->shouldReceive('row_array')->once()->andReturn(NULL);
```

Comment:
Oops! This was testing it actually returned JSON formatted data. :) I'll make another set so that we test both ways now. Thanks.